Index: routes/web.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- routes/web.php	(date 1578831000682)
+++ routes/web.php	(date 1578831000682)
@@ -1,30 +1,46 @@
 <?php
 
+//podstawowe podstrony
 Route::get('/', 'IndexController@index');
 Route::get('/index', 'IndexController@index');
 
-Route::get('/admin', 'AdminController@login');
-Route::post('/admin', 'AdminController@authenticate');
-
 Route::get('/price_list', 'PriceListController@price_list');
 Route::get('/repertoire', 'RepertoireController@repertoire');
 Route::get('/contact', 'ContactController@contact');
 
+//logowanie i rejestracja
+Route::get('session/get','SessionsController@accessSessionData');
+Route::get('session/remove','SessionsController@deleteSessionData');
+
+Route::get('admin/logout','AdminController@deleteSessionData');
+
+Route::get('/register', 'RegistrationController@create');
+Route::post('/register', 'RegistrationController@store');
+
+Route::get('/login', 'SessionsController@login_form');
+Route::post('/login', 'SessionsController@authenticate');
+
+//edycja danych użytkownika
 Route::get('/modify_data', 'IndexController@modify_data');
 Route::post('/modify_data', 'IndexController@modify');
 Route::get('/change_password', 'IndexController@change_password');
 Route::post('/change_password', 'IndexController@change');
 
+//administrator
+Route::get('/admin', 'AdminController@login');
+Route::post('/admin', 'AdminController@authenticate');
+Route::get('/admin/index', 'AdminController@index');
 
-Route::get('/add_hall', 'IndexController@hall');
-Route::post('/add_hall', 'IndexController@add_hall');
+Route::get('/admin/add_hall', 'IndexController@hall');
+Route::post('/admin/add_hall', 'IndexController@add_hall');
 
-Route::get('/add_movie', 'IndexController@movie');
-Route::post('/add_movie', 'IndexController@add_movie');
-Route::post('/remove_movie', 'IndexController@remove_movie');
-Route::get('/add_seance', 'IndexController@seance');
-Route::post('/add_seance', 'IndexController@add_seance');
-Route::post('/remove_seance', 'IndexController@remove_seance');
+Route::get('/admin/add_movie', 'IndexController@movie');
+Route::post('/admin/add_movie', 'IndexController@add_movie');
+Route::post('/admin/remove_movie', 'IndexController@remove_movie');
+
+Route::get('/admin/add_seance', 'IndexController@seance');
+Route::post('/admin/add_seance', 'IndexController@add_seance');
+Route::post('/admin/remove_seance', 'IndexController@remove_seance');
 //Route::get('/index/create', 'IndexController@create');
 
 //Route::get('/index/{page}', 'IndexController@show');
@@ -33,15 +49,4 @@
 
 //Route::post('/index/{page}/reviews', 'IndexController@store');
 
-Route::get('session/get','SessionsController@accessSessionData');
-//Route::get('session/set','SessionsController@storeSessionData');
-Route::get('session/remove','SessionsController@deleteSessionData');
-
 
-Route::get('/register', 'RegistrationController@create');
-Route::post('/register', 'RegistrationController@store');
-//Route::get('/refreshcaptcha', 'RegistrationController@refreshCaptcha');
-
-Route::get('/login', 'SessionsController@login_form');
-Route::post('/login', 'SessionsController@authenticate');
-//Route::get('/logout', 'SessionsController@destroy');
